File: Php/Infrastructure/Data/Classes/RawCashgameResult.php
namespace Infrastructure\Data\Classes {

	use entities\Checkpoints\Checkpoint;

	class RawCashgameResult{

		private $playerId;
		private $checkpoints;

		public function __construct($playerId){
			$this->playerId = $playerId;
			$this->checkpoints = array();
		}

		public function getPlayerId(){
			return $this->playerId;
		}

        public function getCheckpoints(){
            return $this->checkpoints;
        }

		public function addCheckpoint(Checkpoint $checkpoint){
			$this->checkpoints[] = $checkpoint;
		}

	}

}

File: Infrastructure/Data/Classes/RawCashgameResult.cs
using System.Collections.Generic;
using Core.Classes.Checkpoints;

namespace Infrastructure.Data.Classes {

	public class RawCashgameResult{

	    public int PlayerId { get; private set; }
        public List<Checkpoint> Checkpoints { get; private set; }

	    public RawCashgameResult(int playerId)
	    {
	        PlayerId = playerId;
			Checkpoints = new List<Checkpoint>();
	    }

		public void AddCheckpoint(Checkpoint checkpoint){
			Checkpoints.Add(checkpoint);
		}

	}

}